tmt/schemas/results.yaml

```diff
@@ -143,7 +148,7 @@ items:
       $ref: "/definitions/check_results"
 
     subresult:
-      $ref: "/definitions/subresult"
+      $ref: "/definitions/subresults"
```

Comment:
according to the discussion on the mtg, seems they all should start with `#`

Reply:
> I am reading https://json-schema.org/understanding-json-schema/structuring#dollarref and not sure I completely understand why it would be required in some cases, and in some cases not :(

You mean `#`? I think it's implementation specific that it was able to be resolved without. Definitely should change these to `#`, and even `definitions` -> `$defs`

Reply:
WRT `$defs`:

> In drafts 06 and 07 $defs keyword was named definitions, this has changed starting with draft 2019-09. Don’t worry, definitions can still be used (you can use any name, it doesn’t really matter).

Since we're using draft 07, I suppose we should stick to `definitions`. No problem to switch to different draft & rename `definitions`, but that would contain more changes than just the rename.

Reply:
Addressed in 4ddbfc7b